adhocracy4/modules/models.py

```diff
@@ -25,6 +31,42 @@ def settings_instance(self):
                 return getattr(self, setting)
 
     @property
+    def is_active(self):
+        return self.phase_set \
+            .active_phases()\
+            .exists()
+
+    @property
+    def is_active_in_project(self):
+        return self == self.project.active_module
+
+    @property
+    def active_phase(self):
+        return self.phase_set \
+            .active_phases() \
+            .first()
+
+    @functional.cached_property
+    def phases(self):
+        return self.phase_set.all()
+
+    @property
+    def future_phases(self):
+        phases = self.phase_set.filter(models.Q(start_date__gt=timezone.now())
+                                       | models.Q(start_date=None))
```

Comment:
Semantically that was exactly the idea, that non-dated phases are future phases because they are expected to be scheduled.

In SQL comparisons with `Null` are supposed to be `unknown`, which is neither true nor false. So they would not match the select criteria.